docsrc/Doxyfile

```diff
@@ -1405,7 +1405,7 @@ CHM_FILE               =
 HHC_LOCATION           =
 
 # The GENERATE_CHI flag controls if a separate .chi index file is generated
-# (YES) or that it should be included in the master .chm file (NO).
+# (YES) or that it should be included in the main .chm file (NO).
```

Comment:
Semantically I'd change this to something else, maybe "primary" or "root" or whatever

Reply:
went with primary

you monster